Sources/SessionManager/SessionManager+Extension.swift

```diff
@@ -128,7 +128,20 @@ extension SessionManager {
         let newXValue = reversedSharedSecret.hexa
         let hash = SHA2(variant: .sha512).calculate(for: newXValue.hexa).hexa
         let aesEncryptionKey = hash.prefix(64)
```

Comment:
the encrypt use prefix 32 instead of 64 for aes key.
Is this correct here?

Reply:
This is old code which is working. Haven't done any changes over there.